nomos-mix/core/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 cached = "0.53"
 tokio = { version = "1.39" }
 tracing = "0.1"
-rand = "0.8"
+rand = { version = "0.8", features = ["small_rng"] }
```

Comment:
Curious, why small? Probably rand_chacha should be quick and secure enough instead?

Reply:
Yeah, lets do the middle term. It should be easy to change if needed. Thanks!